File: solutions/268.missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
const missingNumber = nums => {
  for (let i = 0; i < nums.length; i++) {
    let num = nums[i]

    if (nums.indexOf(i) === -1) return i
  }

  return nums.length
}

File: solutions/268.missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        actual_len = len(nums) + 1
        s = actual_len * (actual_len - 1) / 2

        return int(s - sum(nums))
